api/src/main/java/net/kyori/adventure/text/BlockNBTComponent.java

```diff
@@ -28,8 +28,16 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * A block NBT component.
+ * A NBT component where the NBT is fetched from an in-game block using a position.
  *
+ * <p>This component consists of:</p>
```

Comment:
We might want to mention it consists of everything from the parent(s)?

Reply:
Yes, this makes sense. I will include the parent values

Reply:
Forgot to mention it in the commit message but I changed this in 13739e7 
Was it something like this you imagined?

Reply:
I was thinking you should mention "and everything in <parent>" or something

Reply:
Listing them all in each spot (parents) may result in some being missed during updates.